extensions/common/vault/vault-aws/src/test/java/org/eclipse/edc/vault/aws/AwsSecretsManagerVaultTest.java

```diff
@@ -63,16 +66,31 @@ void storeSecret_shouldSanitizeKey() {
 
         vault.storeSecret(KEY, value);
 
-        verify(secretClient).createSecret(CreateSecretRequest.builder().name(SANITIZED_KEY)
+        verify(secretClient).updateSecret(UpdateSecretRequest.builder().secretId(SANITIZED_KEY)
                 .secretString(value).build());
     }
 
     @Test
-    void storeSecret_shouldNotOverwriteSecrets() {
-        var value = "value";
+    void storeSecret_shouldUpdateSecretIfExist() {
+        String value = "value";
+
+        vault.storeSecret(KEY, value);
+
+        verify(secretClient).updateSecret(UpdateSecretRequest.builder().secretId(SANITIZED_KEY)
```

Comment:
probably better to use an argument matcher `argThat`.

Reply:
argument matcher isn't used here

Reply:
I'm going to change all the places where there is a validation of how the arguments were passed to use `argThat`